src/app/content/highlights/utils/summaryHighlightsUtils.ts

```diff
@@ -67,6 +69,34 @@ export const addSummaryHighlight = (summaryHighlights: SummaryHighlights, data:
   return newHighlights;
 };
 
+export const getSortedSummeryHighlights =
+  (summaryHighlights: SummaryHighlights | null, locationFilters: HighlightLocationFilters ) => {
```

Comment:
does this function return `SummaryHighlights`? objects have no implicit order in javascript, so if this is supposed to sort things we should really be returning an array